src/Extensions/Script/Serialization/JavaScriptSerializer.cs

```diff
@@ -12,27 +13,29 @@ internal sealed class JavaScriptSerializer
 
     internal static string SerializeInternal(OrderedDictionary orderedDictionary)
     {
-        throw new NotImplementedException();
+        return JsonConvert.SerializeObject(orderedDictionary);
```

Comment:
For now, let's use System.Text.Json. we can make it pluggable and configurable later. I'd rather not bring it in since ASP.NET Core doesn't use it anymore